go.mod

```diff
@@ -15,6 +15,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+replace github.com/hyperledger/firefly-transaction-manager v0.9.10 => github.com/kaleido-io/firefly-transaction-manager v0.0.0-20220902203522-fcba476c9ec5
```

Comment:
Remove once https://github.com/hyperledger/firefly-transaction-manager/pull/33 is merged